README.md

```diff
@@ -105,6 +105,7 @@ export default let BuilderPage = () => {
 &nbsp;&nbsp;useEffect(() => { 
 &nbsp;&nbsp;&nbsp;&nbsp;builder.get('page', { url: '/' })
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;.promise().then(setPage)
+&nbsp;&nbsp;&nbsp;&nbsp;}
 &nbsp;&nbsp;, [])
```

Comment:
Thanks for catching @rojr ! Only tiny nit picky suggestion is to put the bracket here instead

```suggestion
&nbsp;&nbsp;}, [])
```

Reply:
Good shout!

I've updated the PR with this change :+1: